easybuild/easyconfigs/c/Clang-AOMP/Clang-AOMP-5.6.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,76 @@
+name = 'Clang-AOMP'
+_rocm_version = '5.6.0'
+version = _rocm_version
+
+homepage = 'https://github.com/RadeonOpenCompute/llvm-project'
+description = """AOMP is an open source Clang/LLVM based compiler with added
+support for the OpenMP® API on Radeon™ GPUs."""
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Perl', '5.34.1'),
+    ('Python', '3.10.4'),
+    ('elfutils', '0.187'),
```

Comment:
This EasyConfig worked well on my side (thanks for all of this btw). But when loading the installed module I noticed that e.g. the `libhsa-runtime64.so` library links to a few system libraries (`/lib64/libelf.so`, `/lib64/libzstd.so`, etc.) because `elfutils` is only included as a build dependency. Are you seeing this as well? Just moving it to the build-and-runtime dependency list solved it for me.

Reply:
I didn't look that closely at the result, got sidetracked as always...

Reply:
Hmm, where did you see libzstd being used? I can't find it.
Fixing elfutils...